features/step_definitions/bookmark_steps.rb

```diff
@@ -142,6 +141,13 @@
   click_button "Edit"
 end
 
+When /^I show most recent bookmarks on ([^"]*)'s bookmark of "([^"]*)"$/ do |user, title|
+  pseud_id = User.find_by(login: user).default_pseud.id
```

Comment:
You can avoid having to lookup the bookmark ID by grabbing the list element containing both the work title and the bookmarker:

```rb
  within("li.bookmark", text: /#{title}.*Bookmarked by #{user}/) do
    step %{I follow "Show Most Recent Bookmarks"}
  end
```